.github/workflows/_test_build_nix_fake_prerelease.yaml

```diff
@@ -0,0 +1,63 @@
+name: "Test Build Nix fake prerelease"
+
+on:
+  workflow_call:
+    inputs:
+      original_repo_name:
+        required: true
+        type: string
+      repo_full_name:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+      integration:
+        required: true
+        type: string
+    secrets:
+      docker_hub_id:
+        description: "docker hub id"
+        required: true
+      docker_hub_password:
+        description: "docker hub password"
+        required: true
+      gpg_passphrase:
+        description: "GPG passphrase"
+        required: true
+      gpg_private_key_base64:
+        description: "GPG private key base64 encoded"
+        required: true
+
+jobs:
+  test-build-nix:
+    name: Test binary compilation for all platforms:arch
+    runs-on: ubuntu-20.04
+    env:
+      INTEGRATION: ${{ inputs.integration }}
+      ORIGINAL_REPO_NAME: ${{ inputs.original_repo_name }}
+      REPO_FULL_NAME: ${{ inputs.repo_full_name }}
+      TAG: ${{ inputs.tag }}
+      GPG_MAIL: 'infrastructure-eng@newrelic.com'
+      GPG_PASSPHRASE: ${{ secrets.gpg_passphrase }}
+      GPG_PRIVATE_KEY_BASE64: ${{ secrets.gpg_private_key_base64 }}
```

Comment:
is this needed? perhaps using fake ones just for testing? Suggesting it for the sake of reducing inputs and simplify the interface

Reply:
This is needed [here](https://github.com/newrelic/nri-apache/blob/master/build/ci.mk#L81), isn't it?

Reply:
I think is not signed https://github.com/newrelic/nri-apache/blob/master/build/release.mk#L45 so i don't think is needed